gst/tensor_filter/tensor_filter.c

```diff
@@ -228,6 +228,8 @@ static gboolean gst_tensor_filter_stop (GstBaseTransform * trans);
         if (filter->fw && filter->fw->close) \
           filter->fw->close (filter, &filter->privateData); \
         filter->prop.fw_opened = FALSE; \
+        filter->prop.nnfw = _T_F_UNDEFINED; \
+        filter->fw = NULL; \
```

Comment:
👍 

Looks like we need a unit test that stops and restarts a pipeline and another that open fw, close fw, and reopen fw, later...